src/model/enrouter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace TourCMS\OnBoarding\Model;
+
+class enrouter
```

Comment:
```suggestion
class router
```
Good job with this class, its a good idea what you are doing here, but this should not be a model. This could be something the index page could use to route requests